concepticondata/conceptlists/Urban-2016-160.tsv

```diff
@@ -0,0 +1,161 @@
+Urban-2016-160-78	78	car	Artifacts	1111	CAR
+Urban-2016-160-79	79	chair	Artifacts	1476	CHAIR
+Urban-2016-160-80	80	clock	Artifacts	1323	CLOCK
+Urban-2016-160-81	81	glasses	Artifacts	2292	GLASSES
+Urban-2016-160-82	82	house	Artifacts	1252	HOUSE
+Urban-2016-160-83	83	key	Artifacts	1472	KEY
+Urban-2016-160-84	84	knife	Artifacts	1352	KNIFE
+Urban-2016-160-85	85	ladder	Artifacts	1464	LADDER
+Urban-2016-160-86	86	mirror	Artifacts	1171	MIRROR
+Urban-2016-160-87	87	needle	Artifacts	1382	NEEDLE (FOR SEWING)
+Urban-2016-160-88	88	paper	Artifacts	648	PAPER
+Urban-2016-160-89	89	pen	Artifacts	794	PEN
+Urban-2016-160-90	90	rope	Artifacts	1218	ROPE
+Urban-2016-160-91	91	scissors	Artifacts	1328	SCISSORS
+Urban-2016-160-92	92	shoe	Artifacts	1381	SHOE
+Urban-2016-160-93	93	road/street/way	Artifacts	2457	PATH OR ROAD
+Urban-2016-160-94	94	table	Artifacts	1775	TABLE
+Urban-2016-160-95	95	toilet	Artifacts	219	TOILET
+Urban-2016-160-96	96	train	Artifacts	900	TRAIN
+Urban-2016-160-97	97	weapon	Artifacts	951	WEAPONS
+Urban-2016-160-98	98	window	Artifacts	1375	WINDOW
+Urban-2016-160-99	99	Adam‘s apple	Body Parts and Body Fluids	802	ADAM'S APPLE
+Urban-2016-160-100	100	ankle	Body Parts and Body Fluids	803	ANKLE
+Urban-2016-160-101	101	beard	Body Parts and Body Fluids	678	BEARD
+Urban-2016-160-102	102	belly/stomach	Body Parts and Body Fluids	2114	BELLY OR STOMACH
+Urban-2016-160-103	103	bladder	Body Parts and Body Fluids	1746	BLADDER
+Urban-2016-160-104	104	blood	Body Parts and Body Fluids	946	BLOOD
+Urban-2016-160-105	105	bone	Body Parts and Body Fluids	1394	BONE
+Urban-2016-160-106	106	brain	Body Parts and Body Fluids	1583	BRAIN
+Urban-2016-160-107	107	breast	Body Parts and Body Fluids	1402	BREAST
+Urban-2016-160-108	108	buttocks	Body Parts and Body Fluids	834	BUTTOCKS
+Urban-2016-160-109	109	calf	Body Parts and Body Fluids	498	CALF OF LEG
+Urban-2016-160-110	110	cheek	Body Parts and Body Fluids	1730	CHEEK
+Urban-2016-160-111	111	chin	Body Parts and Body Fluids	1510	CHIN
+Urban-2016-160-112	112	eyeball	Body Parts and Body Fluids		
+Urban-2016-160-113	113	eyebrow	Body Parts and Body Fluids	1173	EYEBROW
+Urban-2016-160-114	114	eyelash	Body Parts and Body Fluids	1540	EYELASH
+Urban-2016-160-115	115	eyelid	Body Parts and Body Fluids	17	EYELID
+Urban-2016-160-116	116	finger	Body Parts and Body Fluids	1303	FINGER
+Urban-2016-160-117	117	fingernail	Body Parts and Body Fluids	1258	FINGERNAIL
+Urban-2016-160-118	118	guts	Body Parts and Body Fluids	1334	GUTS
+Urban-2016-160-119	119	heart	Body Parts and Body Fluids	1223	HEART
+Urban-2016-160-120	120	jaw	Body Parts and Body Fluids	798	JAW
+Urban-2016-160-121	121	kidney	Body Parts and Body Fluids	489	KIDNEY
+Urban-2016-160-122	122	lip	Body Parts and Body Fluids	478	LIP
+Urban-2016-160-123	123	liver	Body Parts and Body Fluids	1224	LIVER
+Urban-2016-160-124	124	lungs	Body Parts and Body Fluids	688	LUNG
+Urban-2016-160-125	125	mouth	Body Parts and Body Fluids	674	MOUTH
+Urban-2016-160-126	126	mucus	Body Parts and Body Fluids		
```

Comment:
This is commonly mapped to 208	NASAL MUCUS (SNOT) but I see that "snot" is also in the list. Because "snot" is in brackets, I'd propose to map only "mucus". We might also want to update the mapping in Urban-2011-160 because both glosses are mapped to the same concept.

Reply:
I checked with wikipedia on 'mucus' and it turns out to be a broader term than what is commonly understood by 'snot':

> Mucus serves to protect epithelial cells in the linings of the respiratory, digestive, and urogenital systems, and structures in the visual, and auditory systems from pathogenic fungi, bacteria[2] and viruses. Most of the mucus in the body is produced in the gastrointestinal tract.

So I'd leave this gloss unmapped. And yes, the corresponding mappings in Urban-2011-160 need to be modified.